docs/content/en/getting-started/configuration-markup.md

```diff
@@ -149,6 +150,39 @@ Text
 PlainText
 : The plain variant of the above.
 
+The `render-footnote-link` template will receive this context:
+
+Page
+: The [Page](/variables/page/) being rendered.
+
+Index
+: The footnote numeric index (indicating which number footnote it is in order)
```

Comment:
Is this a zero-based index? (I'd assume so, but since the text content is 1-based I'd want to be sure)